ank/doc/swdesign/plantuml/seq_get_agent.plantuml

```diff
@@ -0,0 +1,47 @@
+@startuml seq_get_agent
+!include ../../../../resources/doc/plantuml/clean.skin
+
+box "Ankaios Ank"
+    actor "user" as user
+    participant Main as main
+    participant "CliCommands" as cli_commands
+    participant "ServerConnection" as server_connection
+    participant "<<task>>\nCommunicationsClient" as communications_client
+end box
+
+box "Ankaios Server"
+    participant "Ankaios Server" as server
+end box
+
+-> main ++ : get agents
+
+== startup ==
+
+... initialization ...
+
```

Comment:
What's done here?

Reply:
It is the same like for the other get commands. I just copied. I think this was simplified to not explicitly mention the argument parsing, tlsconfig setup checks and the init process of the CliCommands. But I have just reused for consistency reasons.

Reply:
Alright, let's leave it as is.